docs/builds/guides/migration/migration-to-27.md

```diff
@@ -1,10 +1,10 @@
 ---
 category: builds-migration
 menu-title: Migration to v27.x
-order: 10
```

Comment:
97?

Reply:
Yes. Then v28 gonna be 96 and then so on and we do not need to rewrite it for like 100 releases, which is about a decade.